Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -221,12 +227,16 @@ def main():
 
     initial_interval = params.get("initial_interval")
     fetch_full_feed = params.get("fetch_full_feed") or False
+    is_incremental_feed = params.get('feedIncremental')
```

Comment:
There is a case that in update, this will return None and then the assert will fail. See: https://github.com/demisto/etc/issues/30318

```suggestion
    is_incremental_feed = params.get('feedIncremental') or False
```